iap-connector.py

```diff
@@ -48,7 +50,14 @@ def GenerateConfig(context):
               }
           }
       }
-  }]
+  }
+
+  if context.properties['network'] is not None:
```

Comment:
Once you fix the default value in schema file, we will run into this error "None is not of type 'string' at ['network']".
How about checking the 'None' value string as a default placeholder in the scheme file?
```suggestion
  if context.properties['network'] != 'None':
```